src/test/java/edu/cornell/kfs/sys/web/mock/CuLocalServerTestBase.java

```diff
@@ -0,0 +1,78 @@
+package edu.cornell.kfs.sys.web.mock;
+
+import java.io.IOException;
+
+import javax.net.ssl.SSLContext;
+
+import org.apache.hc.client5.http.impl.io.PoolingHttpClientConnectionManager;
+import org.apache.hc.core5.function.Decorator;
+import org.apache.hc.core5.http.HttpHost;
+import org.apache.hc.core5.http.URIScheme;
+import org.apache.hc.core5.http.config.Http1Config;
+import org.apache.hc.core5.http.io.HttpServerRequestHandler;
+import org.apache.hc.core5.http.io.SocketConfig;
+import org.apache.hc.core5.http.protocol.HttpProcessor;
+import org.apache.hc.core5.io.CloseMode;
+import org.apache.hc.core5.testing.classic.ClassicTestServer;
+import org.apache.hc.core5.util.Timeout;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.After;
+import org.junit.Before;
+
+public abstract class CuLocalServerTestBase {
+    private static final Logger LOG = LogManager.getLogger();
+
+    protected ClassicTestServer server;
+    protected final URIScheme scheme;
+    protected PoolingHttpClientConnectionManager connManager;
```

Comment:
The connection manager might have only been in use by this class's HTTP client instance, which has now been removed by the latest changes. Should this manager be removed as well?

Reply:
ok